drivers/amazonec2/stub_test.go

```diff
@@ -73,7 +77,9 @@ type fakeEC2WithLogin struct {
 	*fakeEC2
 }
 
-func (f *fakeEC2WithLogin) DescribeAccountAttributes(input *ec2.DescribeAccountAttributesInput) (*ec2.DescribeAccountAttributesOutput, error) {
+func (f *fakeEC2WithLogin) DescribeAccountAttributes(
```

Comment:
Undone.  The 2 `DescribeAccountAttributes` methods are now back to the formatting on master.